### PR TITLE
FIAB Code Snippet

### PR DESCRIPTION
**Pull request by avinashpipalwa**
Created a code snippet for FIAB Calculations

### CODE REVIEW

File: examples/fiab_example.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        startdate = "20200101"
+        enddate = "20201213"
+        fiab_account = FIABIdentifier(id=faccount)
+        fiab_dates = FIABDateParameters(startdate=startdate, enddate=enddate)
+        fiab_calculation_parameters = FIABCalculationParameters(fiabdocument=fiab_document_name, account=fiab_account, dates=fiab_dates)
+    
+        run_calculation = fiab_calculations_api.run_calculation(fiab_calculation_parameters=fiab_calculation_parameters)
+        if run_calculation[1] == 202:
+            calculation_id = run_calculation[2]['X-FactSet-Api-Calculation-Id']
+            print(calculation_id)
```
Comment (vamshimupparaju) [inline examples/fiab_example.py:54]: Minor : Here in print statement just add phrase as 'Calculation Id is'

**Review by ksreeramoj — APPROVED** [review]
LGTM

**Review by vamshimupparaju — APPROVED** [review]
Left one minor comment, Apart from that Looks good to me

**Comment by vamshimupparaju** [comment]
@avinashpipalwa Please update the PR Title as "feat(SDK) : Added FIAD Code Snippet"

File: examples/fiab_example.py (hunk 1 above)
Comment (avinashpipalwa) [reply examples/fiab_example.py]: Added

**Pull request merged**